lib/Wikia/src/Util/Consul/ConsulConfig.php

```diff
@@ -0,0 +1,85 @@
+<?php
+namespace Wikia\Util\Consul;
+
+class ConsulConfig {
+
+	protected $urlSchema = "{tag}.{service}.service.{dataCenter}.consul";
```

Comment:
Tag and data center are optional so perhaps we should construct this with an implode?  In our case, we probably use data center almost everywhere but tag is less frequent.